RDF: Internal Server Error

If you open RDF from someones badge page, you got:
https://badges.fedoraproject.org/user/blaskovic/foaf (Internal Server Error)

FYI, here's the traceback from the server logs:

```python
mod_wsgi (pid=2212): Exception occurred processing WSGI script '/usr/share/tahrir/tahrir.wsgi'.                                                                                                                                    
Traceback (most recent call last):                                                                                                                                                                                                 
  File "/usr/lib/python2.7/site-packages/paste/registry.py", line 379, in __call__                                                                                                                                                 
    app_iter = self.application(environ, start_response)                                                                                                                                                                           
  File "/usr/lib/python2.7/site-packages/moksha/wsgi/middleware/middleware.py", line 76, in __call__                                                                                                                               
    return self.application(environ, start_response)                                                                                                                                                                               
  File "/usr/lib/python2.7/site-packages/moksha/wsgi/middleware/extensionpoint.py", line 232, in __call__                                                                                                                          
    response = request.get_response(self.application)                                                                                                                                                                              
  File "/usr/lib/python2.7/site-packages/WebOb-1.4.1-py2.7.egg/webob/request.py", line 1317, in send                                                                                                                               
    application, catch_exc_info=False)                                                                                                                                                                                             
  File "/usr/lib/python2.7/site-packages/WebOb-1.4.1-py2.7.egg/webob/request.py", line 1281, in call_application                                                                                                                   
    app_iter = application(self.environ, start_response)                                                                                                                                                                           
  File "/usr/lib/python2.7/site-packages/tw2/core/middleware.py", line 204, in __call__                                                                                                                                            
    resp = req.get_response(self.app, catch_exc_info=True)                                                                                                                                                                         
  File "/usr/lib/python2.7/site-packages/WebOb-1.4.1-py2.7.egg/webob/request.py", line 1313, in send                                                                                                                               
    application, catch_exc_info=True)                                                                                                                                                                                              
  File "/usr/lib/python2.7/site-packages/WebOb-1.4.1-py2.7.egg/webob/request.py", line 1281, in call_application                                                                                                                   
    app_iter = application(self.environ, start_response)                                                                                                                                                                           
  File "/usr/lib/python2.7/site-packages/paste/deploy/config.py", line 291, in __call__                                                                                                                                            
    return self.app(environ, start_response)                                                                                                                                                                                       
  File "/usr/lib/python2.7/site-packages/pyramid/router.py", line 242, in __call__                                                                                                                                                 
    response = self.invoke_subrequest(request, use_tweens=True)                                                                                                                                                                    
  File "/usr/lib/python2.7/site-packages/pyramid/router.py", line 217, in invoke_subrequest                                                                                                                                        
    response = handle_request(request)                                                                                                                                                                                             
  File "/usr/lib/python2.7/site-packages/pyramid/tweens.py", line 21, in excview_tween                                                                                                                                             
    response = handler(request)                                                                                                                                                                                                    
  File "/usr/lib/python2.7/site-packages/pyramid_tm/__init__.py", line 82, in tm_tween                                                                                                                                             
    reraise(*exc_info)                                                                                                                                                                                                             
  File "/usr/lib/python2.7/site-packages/pyramid_tm/__init__.py", line 63, in tm_tween                                                                                                                                             
    response = handler(request)                                                                                                                                                                                                    
  File "/usr/lib/python2.7/site-packages/pyramid/router.py", line 163, in handle_request                                                                                                                                           
    response = view_callable(context, request)                                                                                                                                                                                     
  File "/usr/lib/python2.7/site-packages/pyramid/config/views.py", line 385, in viewresult_to_response                                                                                                                             
    result = view(context, request)                                                                                                                                                                                                
  File "/usr/lib/python2.7/site-packages/pyramid/config/views.py", line 501, in _requestonly_view                                                                                                                                  
    response = view(request)                                                                                                                                                                                                       
  File "/usr/lib/python2.7/site-packages/tahrir/views.py", line 921, in user_foaf                                                                                                                                                  
    body = foafutils.generate_foaf_file(user)                                                                                                                                                                                      
  File "/usr/lib/python2.7/site-packages/tahrir/foafutils.py", line 22, in generate_foaf_file                                                                                                                                      
    person = create_foaf_node(graph, FOAF, user)                                                                                                                                                                                   
  File "/usr/lib/python2.7/site-packages/tahrir/foafutils.py", line 34, in create_foaf_node                                                                                                                                        
    graph.add((node, FOAF['img'], r.Namespace(user.avatar_url(512))))                                                                                                                                                              
  File "/usr/lib/python2.7/site-packages/rdflib/graph.py", line 1316, in add                                                                                                                                                       
    _assertnode(s,p,o)                                                                                                                                                                                                             
  File "/usr/lib/python2.7/site-packages/rdflib/graph.py", line 1903, in _assertnode                                                                                                                                               
    'Term %s must be an rdflib term' % (t,)                                                                                                                                                                                        
AssertionError: Term https://seccdn.libravatar.org/avatar/a55b5be6798b4e7faa3710292d0d19d5befa7f0d552a83acb6e154e919a6dc28?s=512&d=https%3A%2F%2Fbadges.fedoraproject.org%2Fstatic%2Fimg%2Fbadger_avatar.png must be an rdflib term
```

--- a/tahrir/foafutils.py
+++ b/tahrir/foafutils.py
@@ -31,7 +31,7 @@ def create_foaf_node(graph, FOAF, user):
     # add the user data
     graph.add((node, r.RDF.type, FOAF['Person']))
     graph.add((node, FOAF['nick'], r.Literal(user.nickname)))
-    graph.add((node, FOAF['img'], r.Namespace(user.avatar_url(512))))
+    graph.add((node, FOAF['img'], r.URIRef(user.avatar_url(512))))
     graph.add((node, FOAF['mbox_sha1sum'], r.Literal(user.email_sha1)))
 
     if user.website:

